recipes/bayes-filters-lib/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "libbayes-filters-lib" %}
+{% set version = "0.10.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/robotology/bayes-filters-lib/archive/refs/tags/{{ version }}.tar.gz
+    sha256: 797ad2285e8f7db3118563a879b9268567f0bd167b7b4a0c446e7f67a6a76451
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage(name, max_pin='x.x.x') }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - pkg-config
+    - ninja
+  host:
+    - eigen
+
+test:
+  commands:
```

Please include check for cmake config files.
```suggestion
  commands:
    - test -f $PREFIX/lib/cmake/BayesFilters/BayesFiltersConfig.cmake  # [unix]
    - if not exist %PREFIX%\\Library\\CMake\\BayesFiltersConfig.cmake  # [win]
```

---

Done.